recipes/gpflow/meta.yaml

```diff
@@ -0,0 +1,45 @@
+{% set version = "1.1.1" %}
+
+package:
+  name: gpflow
+  version: {{ version }}
+
+source:
+  fn: GPflow-{{ version }}.tar.gz
+  url: https://github.com/GPflow/GPflow/archive/{{ version }}.tar.gz
+  sha256: e92737e9488360aca754c24da434118c625aa9737bc22d2a58e41caf7c3f9122
+
+build:
+  number: 0
+  skip: True  # [py<35]
```

Comment:
So, this could be `noarch: python`, by changing the requirements to `python >=3.5` below (https://conda-forge.org/docs/meta.html#building-noarch-packages). That would mean we wouldn't actually be able to run an import test at all, since it would only build on Linux; maybe that's worth it anyway? (Could add a file-existence test that it puts something in `site-packages`....)

Reply:
I prefer to have a test here then than to save ci time.